srcalc/src/main/java/gov/va/med/srcalc/util/RetrievedValue.java

```diff
@@ -0,0 +1,51 @@
+package gov.va.med.srcalc.util;
```

Comment:
I'm not as sure about this one, but I think another package may be more fitting here.